manifests/k/KurtZimmermann/EventLogViewer/1.015/KurtZimmermann.EventLogViewer.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: KurtZimmermann.EventLogViewer
+PackageVersion: "1.015"

```

```suggestion
PackageVersion: "1.014"
```

After installation, I'm getting the `1.014` version. Could you check the version again?

---

It's version 1.014. Can you correct it or should I submit everything again?